metsim/metsim.py

```diff
@@ -74,6 +66,12 @@ class MetSim(object):
         "base_elev": 0,
         "t_max_lr": '',
         "t_min_lr": '',
+        "sw_prec_thresh": 0.0,
+        "mtclim_swe_corr": False,
+        "lw_cloud": 'cloud_deardorff',
+        "lw_type": 'prata',
+        "tdew_tol": 1e-3,
+        "tmax_daylength_fraction": 0.67,
```

Comment:
What's the rationale for removing these values from `constants.py` and hardcoding them here? I'd rather that they are all collected in one place and then referenced here.